MonstieWash/Assets/Scripts/MonsterAi/MoodArea.cs

```diff
@@ -113,6 +116,8 @@ private void ApplyTouch()
         {
             m_monsterBrain.UpdateMood(moodEffect.reactionStrength * (currentEffectiveness/100f), moodEffect.mood);
             if (debug) print($"Reaction Strength  {moodEffect.reactionStrength}  at effectivness of {currentEffectiveness} for the mood {moodEffect.mood.MoodName}");
+
+            m_moodFXManager.MoodParticleSystems[moodEffect.mood].Play();
```

Comment:
Could add a null check as will throw errors if mood isn't in dict, (although this should never be the case if we set it up right, im just nitpicking cos your code is too damn clean )